lazy.ansible/.manala/docker/Dockerfile.tmpl

```
@@ -76,6 +78,11 @@ RUN \
         pipx \
         python3-argcomplete \
         shellcheck \
+    {{- if $apt.packages }}
+        {{- range $package := $apt.packages }}
+        {{ $package }} \
+        {{- end }}
+    {{- end }}
```

A `if` is not really necessary, `packages` is an empty array by default, if there are no packages, loop content is not involved
```suggestion
        {{- range $package := $apt.packages }}
        {{ $package }} \
        {{- end }}
```